.meta/sources/journald.toml

```diff
@@ -2,6 +2,7 @@
 beta = true
 common = true
 delivery_guarantee = "best_effort"
+except_operating_systems = ["windows"]
```

Comment:
For `journald` it should be `only_operating_systems = ["linux"]`, because it is not supported on macOS.